lib/ansible/modules/find.py

```diff
@@ -434,6 +435,8 @@ def statinfo(st):
 
 
 def handle_walk_errors(e):
+    if e.errno in (errno.EPERM, errno.EACCES):
+        return
```

Comment:
I think we need tests that the return value `skipped_paths` is correct and also we don't abort the search on permission errors. Just ignoring these would be a regression for https://github.com/ansible/ansible/issues/25314.